.babelrc.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  presets: ['@theforeman/builder/babel'],
+  presets: ['./webpack/babel'],
```

Comment:
Does this need to be relative to the file itself using `__dirname`?

Reply:
Need for it to work? probably not, but for consistency we can